### ISSUE TITLE
Cannot read property 

### ISSUE THREAD

**Issue by Kuroiteiken** [mettalogic/arcanum-automation#5]
userscript.html?id=883c06ba-b8f0-4e1e-a19f-510c1fc81696:433 Uncaught TypeError: Cannot read property 'innerHTML' of undefined
    at tc_automate (userscript.html?id=883c06ba-b8f0-4e1e-a19f-510c1fc81696:433)
    at Window.eval (userscript.html?id=883c06ba-b8f0-4e1e-a19f-510c1fc81696:1379)
    at <anonymous>:2:494
    at c.<computed> (eval at exec_fn (www.lerpinglemur.com/:1), <anonymous>:54:182)
tc_automate @ userscript.html?id=883c06ba-b8f0-4e1e-a19f-510c1fc81696:433
eval @ userscript.html?id=883c06ba-b8f0-4e1e-a19f-510c1fc81696:1379
(anonymous) @ VM2289:2
c.<computed> @ VM2298:54

**Comment by mettalogic** [comment]
Yeah, I ought to fix that, but it shouldn't cause any problems. It comes from the sublimate lore popup confirmation code, as there are two other DOM elements that match the class selector, but they don't have the requisite innerHTML property.
